config.toml.dist

```diff
@@ -51,3 +51,10 @@ secret =
 discovery_url = "https://accounts.google.com/.well-known/openid-configuration"
 # Google OpenID Connect id_token issuer domain - Default: "accounts.google.com"
 issuer_domain = "accounts.google.com"
+
+
+[limits]
+# Limit the number of session of any kind.
+auth = ["per-second:3:1:1", "per-minute:20:60:5"]
+# Limit the number of email loop authentications.
+auth_email = ["per-second:1:1:1", "per-minute:10:60:5"]
```

Comment:
I'm starting to think simpler is probably better too.

Calculating granularity is something I tried, but troublesome while we enforce `timespan % granularity == 0`. Something that could be worked around.

I think, as long as we use small windows, something like `10/min` per email with a single key could work just fine.